### PR TITLE
docs: fixup Bitbucket workflows

### PR DESCRIPTION
**Pull request by casperdcl**
- [x] depends on #354
- [x] depends on #355
- [x] depends on #356
- [x] ~depends on #357~
- [x] minor Bitbucket fixup

### CODE REVIEW

**Review by 0x2b3bfa0 — COMMENTED** [review]
Same as https://github.com/iterative/cml.dev/pull/357#pullrequestreview-1151445753

File: content/docs/usage.md [hunk 1]
```diff
@@ -117,13 +117,13 @@ pipelines:
...
+          - python train.py # generate plot.png
     - step:
         name: Create CML report
         script:
-          - cat metrics.txt > report.md
-          - echo >> report.md
-          - echo '![](./plot.png)' >> report.md
+          # Post reports as comments in Bitbucket
+          - cat metrics.txt >> report.md
+          - echo '![](./plot.png "Confusion Matrix")' >> report.md
```
Comment (0x2b3bfa0) [inline content/docs/usage.md:126]: Unnecessarily verbose demonstration of arcane Markdown syntax?

**Review by 0x2b3bfa0 — APPROVED** [review]

File: content/docs/usage.md (hunk 1 above)
Comment (casperdcl) [reply content/docs/usage.md:126]: Perhaps; what does @iterative/cml think?

Comment (0x2b3bfa0) [reply content/docs/usage.md:126]: The same it thought on October 23rd. 😉 Let's ask [the rest of] @iterative/cml.

**Pull request merged**